### ISSUE TITLE
Is the picture specific？

### ISSUE THREAD

**Issue by lijie121** [bmorris3/shampoo#36]
when i try run the demo use fro my picture, the error is coming. of course, I have modified some parameters. for example,  wavelength、dx and dy ...
so, Can you provide some photo collections? or explain the specific format of the input picture.
thank you very much，Your code has a great help for me .

**Comment by bmorris3** [comment]
Could you provide the example code that's not working? It's hard to debug without seeing the code.

**Comment by lijie121** [comment]
this is my code:

    from shampoo import Hologram
    import matplotlib.pyplot as plt
    holograme_path = '/home/lijie/paper/shampoo/data/test.tif'
    propagation_distace = 0.008
    h= Hologram.from_tif(holograme_path,wavelength=650e-6, dx=3.5e-6, dy=3.5e-6)
    wave = h.reconstruct(propagation_distace)
    fig, ax = wave.plot()
    fig.suptitle("USAF Target")
    fig.tight_layout()
    plt.show()

When I run the code will appear two warning, 
![image](https://user-images.githubusercontent.com/11783328/38005782-dfe7ff3e-3274-11e8-9b59-e85977863c98.png)
The first warning is related to the input picture，specific format of the input picture？
The second warning does not know what the reason but often occurs, causing the program to die.
thank you

**Comment by lijie121** [comment]
![image](https://user-images.githubusercontent.com/11783328/38005913-82c61434-3275-11e8-8bfa-586bd2d9dc7c.png)

**Comment by bmorris3** [comment]
The Runtimewarning is pretty common, and not actually a problem. The UserWarning is not common – it means that the `PIL` or `scikit-image` can't read your TIF file. Are you sure it's a valid TIF file?

**Comment by lijie121** [comment]
yes, i think my picture is invalid, so i want to know the specific requirements of the tif file format, for example Bit depth, and ...

**Comment by bmorris3** [comment]
It should be a file that can be opened with [this function](http://scikit-image.org/docs/dev/api/skimage.io.html#skimage.io.imread).
